R/mlRegressionNeuralNetwork.R

```diff
@@ -0,0 +1,755 @@
+#
+# Copyright (C) 2017 University of Amsterdam
```

Comment:
I have not looked into this file, but I think that my comments will be pretty similar to that of the classification one.